pkg/sidecarterminator/terminator.go

```diff
@@ -4,35 +4,37 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"io/ioutil"
 	"strconv"
 	"strings"
 	"syscall"
 	"time"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/remotecommand"
 	"k8s.io/klog"
 )
 
+const (
+	SIDECAR_TERMINATOR_CONTAINER = "sidecar-terminator"
```

Comment:
```suggestion
	SidecarTerminatorContainerNamePrefix = "sidecar-terminator"
```

The Go standard is for camel case names on consts. Also might be good to clarify it's part of the name prefix.

Reply:
Thanks! Duh on my part!